resources/metadata/fortify.yaml

```diff
@@ -0,0 +1,442 @@
+      - name: buildDescriptor
+        type: stash
+      - name: deployDescriptor
+        type: stash
+      - name: tests
+        type: stash
+      - name: opensourceConfiguration
+        type: stash
+    params:
+    - name: authToken
+      type: string
+      description: The FortifyToken to use for authentication
+      mandatory: true
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: autoCreate
+      type: bool
+      description: Whether Fortify project and project version shall be implicitly auto created in case they cannot be found in the backend 
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: mvnCustomArgs
+      type: string
+      description: Allows providing additional Maven command line parameters
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: ''
+    - name: modulePath
+      type: string
+      description: Allows providing the path for the module to scan
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: './'
+    - name: pythonRequirementsFile
+      type: string
+      description: 'The requirements file used in `scanType: ''pip''` to populate
+        the build environment with the necessary dependencies'
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: autodetectClasspath
+      type: bool
+      description: Whether the classpath is automatically determined via build tool i.e. maven or pip or not at all
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: pythonRequirementsInstallSuffix
+      type: string
+      description: 'The suffix for the command used to install the requirements file in `scanType: ''pip''` to populate
+        the build environment with the necessary dependencies'
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: pythonVersion
+      type: string
+      description: 'Python version to be used in `scanType: ''pip''`'
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: python3
+    - name: uploadResults
+      type: bool
+      description: Whether results shall be uploaded or not
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: buildDescriptorFile
+      type: string
+      description: 'Path to the build descriptor file addressing the module/folder
+        to be scanned. Defaults are for scanType=`maven`: `./pom.xml`, scanType=`pip`:
+        `./setup.py`, scanType=`mta`: determined automatically'
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: commitId
+      description: 'Set the Git commit ID for identifing artifacts throughout the scan.'
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: git/commitId
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      type: string
+    - name: commitMessage
```

Comment:
can we have a default here? Maybe even a fixed value without the necessity to configure it?
Maybe I also misunderstand the description?